README.md

```diff
@@ -47,6 +49,8 @@ Ruby | 🚧
 * [Configuration](#Configuration-⚙️)
   * [Practices](#Practices)
   * [GitHub CI Action](#GitHub-Ci-Action)
+* [Support](#Support-🦸‍♀️-🦸‍♂️)
+* [Contributing](#Contributing-👩‍💻-👨‍💻)  
```

Comment:
It doesn't work. See https://github.com/DXHeroes/dx-scanner/tree/docs/add-slack-invite